serpentTools/objects/xsdata.py

```diff
@@ -114,3 +131,39 @@ def hasExpectedData(self):
             return False
 
         return True
+
+    def plot(self, mts, **kwargs):
```

Comment:
I would recommend looking at some of the other plot methods we have and try and implement a *similar* approach, mainly with some of the formatting arguments like autolabeling and axis scaling. The [magicPlotDecorator](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/develop/serpentTools/plot.py#L62) may be a useful guide for

1. a consistent set of arguments and,
2. minimal docstring writing for various kwargs :+1:

Reply:
Thanks, will do, Drew!